src/chaplin/views/view.coffee

```diff
@@ -194,7 +194,7 @@ module.exports = class View extends Backbone.View
   #   e.g.
   #   @delegate('click', 'button.confirm', @confirm)
   delegate: (eventName, second, third) ->
-    if Backbone.utils
+    if Backbone.utils and Backbone.utils?.delegate
```

Comment:
**Conclusion**

Without determining whether `Backbone.utils` **exists** and `Backbone.utils.delegate`/`Backbone.utils.undelegate` are **functions**, any call of `Chaplin.delegate` or `Chaplin.undelegate` results in an error for me:


> ```sh
Backbone.utils.delegate is undefined
> ```

or 

> ```sh
Backbone.utils.undelegate is undefined
> ```

The `Backbone.utils` object exists - **but** without `delegate` and `undelegate` methods. 

Please check the source: https://github.com/paulmillr/exoskeleton/blob/master/lib/utils.js

The latest release of exoskeleton does not support this methods (*possibly deprecated?*).

As already mentioned above, I want to use `Chaplin` with the latest release of exoskeleton. This could be a possible fix for paulmillr/exoskeleton#87, isn't it?

And as far I understand, `Chaplin` seems to have its own, native `delegate` and `undelegate` methods:

`View.delegate`: [ chaplin/src/chaplin/views/view.coffee#L202-L225](https://github.com/chaplinjs/chaplin/blob/master/src/chaplin/views/view.coffee#L202-L225)
`View.undelegate`: [ chaplin/src/chaplin/views/view.coffee#L260-L276](https://github.com/chaplinjs/chaplin/blob/master/src/chaplin/views/view.coffee#L260-L276)

Please correct me, if I am wrong.